lib/registry/src/lib.rs

```diff
@@ -317,6 +318,7 @@ pub fn query_command_from_registry(
     let package = command.package_version.package.display_name;
     let version = command.package_version.version;
     let url = command.package_version.distribution.download_url;
+    let pirita_url = command.package_version.distribution.pirita_download_url;
 
     Ok(PackageDownloadInfo {
```

Comment:
Why don't we return the type/content from `command.package_version`. That seems way easier than recreating a new struct each time

Reply:
I'll approve in any case, we can revisit this after

Reply:
Yeah, you have a good point.

Something we should do is make all our GraphQL queries public. That way people can execute them directly if a high-level function (e.g. `wasmer_registry::query_command_from_registry()`) doesn't support exactly what you are looking for.